Assets/Scripts/Vision/AnimalVision.cs

```diff
@@ -0,0 +1,134 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+#if UNITY_EDITOR
+using UnityEditor;
+using UnityEditorInternal;
+#endif
+using UnityEngine;
+using Anivision.Core;
+
+namespace Anivision.Vision
+{
+    /// <summary>
+    /// Script that contains the vision information for an animal
+    /// Developers/Artists can check various parameters through the inspector
+    /// </summary>
+    public class AnimalVision : MonoBehaviour
+    {
+        public Colorblindness colorblindness = Colorblindness.None;
+        public List<VisionEffect> effects;
```

Comment:
For modularity's sake, shouldn't colorblindness be a vision effect?
Also, if we're doing custom vision matrices, colorblindness might not even be the best term for it, as we're not necessarily losing colors, only shifting them.

Reply:
Yep, if you look at the VisionEffect enum, Colorblindness is a type of VisionEffect. However, the `Colorblindness` refers to the type of colorblindness. It is kind of confusing now that I think about it. Should I rename `Colorblindness` to `ColorblindType`? Also not sure I understand the thing about the custom vision matrices

Reply:
I think changing to ColorblindType is a good idea. What I meant by custom vision matrices is that using them, we don't necessarily lose color information. We can move red to blue, and blue to red, for example, so we move information, but don't lose it. In cases like these, were aren't really "blind" to any colors, they've just moved.

Reply:
Hmmm, I agree, colorblindness isn't the most accurate description, but I do wonder, because it's such a widely used term, if we should keep it that? Or should we change it to ColorShift? But then that might be confusing for things like UV or other false color effects, where we **actually** shift the channels. Looking at photoshop, they call it a Channel Mixer. But is it more confusing to call it, for example, `ChannelMix.Protanopia`, as opposed to `Colorblind.Protanopia` for someone who wants to apply the protanopia effect?

Reply:
Ok, I'm fine with keeping it as colorblind